openpaas-james/apps/distributed-es6-backport/src/main/java/com/linagora/openpaas/james/app/DistributedServer.java

```diff
@@ -108,6 +119,7 @@
 
     public static final Module PROTOCOLS = Modules.combine(
         JMAP,
+        new KeystoreModule(),
```

Comment:
That `KeystoreModule` is binded to the `InMemoryKeystoreManager` I think?

Reply:
Yes, and I will solve it as soon as you wrote the tests for the CassandraKeystoreManager

Reply:
#128